snapcraft/internal/repo.py

```diff
@@ -527,3 +530,26 @@ def _try_copy_local(path, target):
 def check_for_command(command):
     if not shutil.which(command):
         raise MissingCommandError([command])
+
+
+def _get_pkg_name_parts(pkg_name):
+    """Break package name into base parts"""
+
+    name = pkg_name
+    version = arch = None
+    if '=' in pkg_name:
+        name, version = pkg_name.split('=')
+        if ':' in name:
+            name, arch = name.split(':')
+
+    if arch:
```

Comment:
Thanks for catching that.  It was a holdover from my mistaken thinking that the format was <pkg>=<version>:<arch>  I'm testing an update now.